openpype/hosts/equalizer/addon.py

```diff
@@ -0,0 +1,33 @@
+import os
+from openpype.modules import OpenPypeModule, IHostAddon
+
+EQUALIZER_HOST_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+class EqualizerAddon(OpenPypeModule, IHostAddon):
+    name = "equalizer"
+    host_name = "equalizer"
+
+    def initialize(self, module_settings):
+        self.enabled = True
+
+    def add_implementation_envs(self, env, _app):
+        # 3dequalizer utilize PYTHON_CUSTOM_SCRIPTS_3DE4 for custom scripts
+        # and TDE4_ROOT for its root directory. Note that the custom script
+        # paths are semicolon separated even on Windows.
```

Comment:
Default on Windows is actually semi-colon separated `;` - did you mean that on windows it's **colon** `:` separated?
```suggestion
        # paths are colon `:` separated even on Windows.
```